eventuate-core/src/main/scala/com/rbmhtechnology/eventuate/ReplicationEndpoint.scala

```diff
@@ -363,6 +359,96 @@ class ReplicationEndpoint(
 }
 
 /**
+ * An [[EndpointFilter]] computes a [[ReplicationFilter]] that shall be applied to a
+ * replication read request that replicates from a source log (defined by ``sourceLogName``)
+ * to a target log (defined by ``targetLogId``).
+ */
+trait EndpointFilter {
+  def filterFor(targetLogId: String, sourceLogName: String): ReplicationFilter
+}
+
+object EndpointFilter {
```

Comment:
I talked about both, the trait and companion object. Anyway, `EndpointFilters` is fine for me.